### ISSUE TITLE
Error after SVG export

### ISSUE THREAD

**Issue by osipxd** [RedMadRobot/figma-export#43]
This error occurs after the export of any SVG. I've not tested it for WEBP
```
2020-11-27T09:13:21+0300 info: Using FigmaExport to export images to Android Studio project.
2020-11-27T09:13:21+0300 info: Fetching images info from Figma. Please wait...
2020-11-27T09:13:25+0300 info: Processing images...
2020-11-27T09:13:25+0300 info: Downloading remote files...
2020-11-27T09:13:26+0300 info: Downloaded 1/1
2020-11-27T09:13:26+0300 info: Converting SVGs to XMLs...
-c parsed, so we will convert the SVG files
-in parsed /var/folders/3c/nnj_p6v93bg7pgs4dkm6cgzc0000gp/T/596D7CCE-4712-4980-9691-7AEC3B68B944
Convert 1 SVG files in total, errors found in 0 files
2020-11-27T09:13:31+0300 info: Writting files to Android Studio project...
Error: Error Domain=NSCocoaErrorDomain Code=4 "“76A43668-8283-4F87-935F-849B32DD57D7” couldn’t be removed." UserInfo={NSUserStringVariant=(
    Remove
), NSFilePath=/var/folders/3c/nnj_p6v93bg7pgs4dkm6cgzc0000gp/T/76A43668-8283-4F87-935F-849B32DD57D7, NSUnderlyingError=0x7fc9e191f630 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
```

**Comment by subdan** [comment]
Thanks. I will fix it by the end of the week.

**Comment by subdan** [comment]
Which version of FigmaExport do you use?

**Comment by osipxd** [comment]
Oh, I was using 0.17.0. Thanks!
Another idea - add a command to check figma-export version :D
Something like `figma-export --version`

**Comment by osipxd** [comment]
I've created a separate issue for the `--version` flag - #45

**Comment by osipxd** [comment]
Hmm... The problem still here on 0.18.3
```
$ figma-export images "280x280 / Estimate / Supernova / *"
2020-11-27T18:35:09+0300 info com.redmadrobot.figma-export : Using FigmaExport to export images to Android Studio project.
2020-11-27T18:35:09+0300 info com.redmadrobot.figma-export : Fetching images info from Figma. Please wait...
2020-11-27T18:35:14+0300 info com.redmadrobot.figma-export : Processing images...
2020-11-27T18:35:14+0300 info com.redmadrobot.figma-export : Downloading remote files...
2020-11-27T18:35:15+0300 info com.redmadrobot.figma-export.file-downloader : Downloaded 1/5
2020-11-27T18:35:16+0300 info com.redmadrobot.figma-export.file-downloader : Downloaded 2/5
2020-11-27T18:35:16+0300 info com.redmadrobot.figma-export.file-downloader : Downloaded 3/5
2020-11-27T18:35:16+0300 info com.redmadrobot.figma-export.file-downloader : Downloaded 4/5
2020-11-27T18:35:16+0300 info com.redmadrobot.figma-export.file-downloader : Downloaded 5/5
2020-11-27T18:35:16+0300 info com.redmadrobot.figma-export : Converting SVGs to XMLs...
2020-11-27T18:35:20+0300 info com.redmadrobot.figma-export.vector-drawable-converter : Convert 5 SVG files in total, errors found in 0 files
2020-11-27T18:35:20+0300 info com.redmadrobot.figma-export : Writting files to Android Studio project...
Error: Error Domain=NSCocoaErrorDomain Code=4 "“B00E3DFF-20D8-4BA2-BFC0-F3350225234C” couldn’t be removed." UserInfo={NSUserStringVariant=(
    Remove
), NSFilePath=/var/folders/3c/nnj_p6v93bg7pgs4dkm6cgzc0000gp/T/B00E3DFF-20D8-4BA2-BFC0-F3350225234C, NSUnderlyingError=0x7ffa186473d0 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}
```
What info should I provide to help fix the issue?

**Comment by subdan** [comment]
Step 1. Try to execute similar command `figma-export images "img/zero/*"` for example android project which you can find on `Examples/AndroidExample`. Do you have the same issue? If no — go to step 2.

Step 2. Could you give me access to your figma file via slack? I want to try to export this image on my machine.

**Comment by greatmatis** [comment]
I having the same issue. Tried Step 1 from comment above. Same error message.

**Comment by subdan** [comment]
Fixed in [0.18.4](https://github.com/RedMadRobot/figma-export/releases/tag/0.18.4). Please check.

**Comment by osipxd** [comment]
Fixed for me
